### ISSUE TITLE
feat: add complexity to this POC

### ISSUE THREAD

**Issue by gioboa** [gioboa/react-microfrontend-demo#7]
To show the potential of the Micro Frontend architecture and how to use @module-federation/vite, it would be nice to create a more structured example with various use cases, various components, routes and external libraries too.

-----

I think we can proceed by steps. A good design is at the base of a nice POC, then we can add more functionalities and we can iterate on it.

**Comment by kandros** [comment]
My 2 cents, I think this is already a great "getting started" with the right level of complexity. It shows you that all you need to get started is relatively small, you can read all the code in 15 min.

Seeing how small it was got me interested enough to say "Let's play a bit with it right now" instead of saving it for later (and never open it again 😅)

A separate more complex project is definitely welcome

**Comment by gioboa** [comment]
Yep, you are right, that was the initial intention. A small project with only few files, easy to discover. On the other hand a complex app can attract big corporations, maybe a separate branch could be the solution.
